### DISCUSSION TITLE
Product Alerts - new information we should store in database

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1680] [Ideas]
As they are right now the product_alert_price and product_alert_stock tables store only a few information about a subscriber and some important are missing. With the new GDPR rules I would suggest a change in the way OpenMage is dealing with product alerts subscriptions by adding in tables new columns like: IP, browser information, remove date. Comparing these tables with newsletter_subscriber they have more information stored which is much better than nothing.

Let's say someone complains he unsubscribed from a product alert but continue to receive notifications. Having this information stored is easy to prove when the subscription was done. 

![a](https://user-images.githubusercontent.com/8360474/120904813-dea87080-c656-11eb-957b-cac6bb77f9d7.jpg)

**Comment by elidrissidev** [comment]
This sounds interesting, I think it's better if it's put in an Issue. I might give it a try in the future.
